eZ/Publish/Core/Persistence/Legacy/Content/Type/Gateway/DoctrineDatabase.php

```diff
@@ -693,6 +694,9 @@ protected function setCommonFieldColumns(
         )->set(
             $this->dbHandler->quoteColumn('can_translate'),
             $q->bindValue(($fieldDefinition->isTranslatable ? 1 : 0), null, \PDO::PARAM_INT)
+        )->set(
+            $this->dbHandler->quoteColumn('is_thumbnail'),
+            $q->bindValue(($fieldDefinition->isThumbnail ? 1 : 0), null, \PDO::PARAM_INT)
```

Comment:
Following previous comment:
```suggestion
            $q->bindValue($fieldDefinition->isThumbnail, null, \PDO::PARAM_BOOL)
```